astropy/coordinates/angle_utilities.py

```diff
@@ -155,13 +170,18 @@ def t_error(t):
 
         def p_angle(p):
             '''
-            angle : hms
-                  | dms
-                  | arcsecond
-                  | arcminute
-                  | simple
+            angle : sign hms eastwest
```

Comment:
Why is it once `eastwest` and the rest only dir?

Reply:
@MaxNoe - since I added that: the logic is that for the `hms` case we know the number is in hours, and thus this must be a longitude and therefore the only directions possible are east and west (a change of 1hN is meaningless; what is an hour in latitude?).

(Ideally, I'd preclude using NS on anything with units of hourangle, but that is hard...)

Reply:
its because we know the units are hours, then only E or W is allowed.

Reply:
Ah, sure makes sense.